bip-0009/assignments.mediawiki

```diff
@@ -0,0 +1,37 @@
+==Deployments==
+
+List of proposed deployments.
+
+State can be defined, active, failed. Dates are in UTC.
+
+{| class="wikitable"
```

Comment:
Luke was suggesting to sort the table and the best way to do this would be
dynamic. (i.e. adding the sortable keyword). The actual mediawiki parser
understands sortable but the github parser does not, I think.